CONTRIBUTING.md

```diff
@@ -12,14 +12,12 @@ This is the process for committing code to the Scala project. There are of cours
     2. The ticket has been discussed and prioritized by the team.
 3. You should always perform your work in its own Git branch. The branch should be given a descriptive name that explains its intent. Some teams also like adding the ticket number and/or the [GitHub](http://github.com) user ID to the branch name, these details is up to each of the individual teams. (See below for more details on branch naming.)
 4. When the feature or fix is completed you should open a [Pull Request](https://help.github.com/articles/using-pull-requests) on GitHub.
-5. The Pull Request should be reviewed by other maintainers (as many as feasible/practical). Note that a reviewer can also be an outside contributor—members of Typesafe and independent contributors are encouraged to participate in the review process. It is not a closed process. Please try to avoid conflict of interest—the spirit of the review process is to evenly distribute the understanding of our code base across its maintainers as well as to load balance quality assurance. Assigning a review to a "sure win" reviewer is not a good long-term solution.
+5. The Pull Request should be reviewed by other maintainers (as many as feasible/practical). Note that a reviewer can also be an outside contributor—members of Typesafe  or VirtusLab and independent contributors are encouraged to participate in the review process. It is not a closed process. Please try to avoid conflict of interest—the spirit of the review process is to evenly distribute the understanding of our code base across its maintainers as well as to load balance quality assurance. Assigning a review to a "sure win" reviewer is not a good long-term solution.
 6. After the review, you should resolve issues brought up by the reviewers as needed (pushing a new commit to address reviewers' comments), iterating until the reviewers give their thumbs up, the "LGTM" (acronym for "Looks Good To Me").
 7. Once the code has passed review the Pull Request can be merged into the distribution.
 
 ## Pull Request Requirements
 
-First, please have a look at and follow the [Pull Request Policy](https://github.com/scala/scala/wiki/Pull-Request-Policy) for guidelines on submitting a pull request to the dotty project. (the pull request policy is the same as for the Scala programming language)
```

Comment:
From what I can tell, this doesn't exist anymore as the link just now redirects to the Scala github. Has this been moved somewhere else? If not I just removed it.

Reply:
@SethTisue maybe you know what has happened with https://github.com/scala/scala/wiki/Pull-Request-Policy ?

Reply:
scala/scala hasn't had a wiki in yeeeeeeeeeears, so that link must be super ancient

we merged the content into README.md and/or CONTRIBUTING.md, e.g at scala/scala@197845620cb827209272bb02e6740feac9f73a48

but in a Scala 3 context, it probably doesn't make sense to direct anybody to either of those files wholesale. but feel free to copy-and-paste anything you'd like to steal

Reply:
If there's no objection, I could merge this PR as it is and copy some interesting paragraphs from scala/scala later

Reply:
Sure! That's fine by me.